codegen/datamodelgen.py

```diff
@@ -81,15 +81,17 @@ def _build_command_docstring(name: str, info: Any, indent: str):
 
 
 class DataModelStaticInfo:
-    def __init__(self, rules: str, mode: str):
+    def __init__(self, rules: str, mode: str, rules_save_name: str = ""):
```

Comment:
Different datamodel for icing/aero or plain flserver datamodel. 
The rules_save_name permits to store out to different datamodels for the same remote structure.